scripts/gen_dockerignore.sh

```diff
@@ -1,2 +1,5 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Why use bash here instead of sh? With the ``cd ..`` it is still POSIX shell no need to require bash.